phpBB/phpbb/cron/task/wrapper.php

```diff
@@ -105,6 +112,25 @@ public function get_url()
 		return $this->routing_helper->route('phpbb_cron_run', $params);
 	}
 
+	/**
+	 * Returns HTML for an invisible `img` tag that can be displayed on page
+	 * load to trigger a request to the relevant cron task endpoint.
+	 *
+	 * @return string       HTML to render to trigger cron task
+	 */
+	public function get_html_tag()
+	{
+		$this->template->set_filenames([
+			'cron_html_tag' => 'cron.html',
+		]);
+
+		$this->template->assign_vars([
```

Comment:
Oops, here we should use `assign_var` because only one var is being instantiated, sorry,

Reply:
`$this->template->assign_var('CRON_TASK_URL', $this->get_url());` should do.